.travis.yml

```diff
@@ -13,7 +13,7 @@ matrix:
         - git config --global user.name "${GH_NAME}"
         - git config --global user.email "${GH_EMAIL}"
         - echo "machine github.com login ${GH_NAME} password ${GH_TOKEN}" > ~/.netrc
-        - cd docs/site && npm install && GIT_USER="${GH_USER}" npm run publish-gh-pages
+        - cd docs/site && npm install && GIT_USER="${GH_NAME}" npm run publish-gh-pages
```

Comment:
It might be a little difference here, I guess.  `GH_USER` env var is used by `npm run publish-gh-pages` for linking repo, i.e. `dsngroup` for `dsngroup/broke`. Therefore, we need a mechanism that is different from user name, a.k.a `GH_NAME`; otherwise, we may publish the site to the fork, i.e. `tz70s/broke`?

You can try it on your fork repo/travis for testing.

Reply:
<img width="836" alt="image" src="https://user-images.githubusercontent.com/13215329/49558569-76258f00-f946-11e8-8545-9f9b5849862f.png">
But how do you setup these environmental variables before? I just modified them to my own settings so that travis can do ```git push``` to ```dsngroup/broke```.

Reply:
I think I can't create a token to an organization account like our ```dsngroup```

Reply:
My previous setting,

```bash
GH_NAME=tz70s
GH_USER=dsngroup
```

Did this autogenerate site yet? If yes, we can merge this.

Reply:
I haven't tested this yet. Let me try it first.

Reply:
This works. However, the ```publish-gh-pages``` script pushes the site to ```gh-pages``` branch, which is different from our current setting (place static files in ```/doc```, including ```javadoc```).  Is there any good idea?

Reply:
Nice, I'd like to use `gh-pages` and remove generated doc in master branch. We can change the serving url path, and further auto-generate java doc like this as well!